### PR TITLE
Merge `xcode10.2` into `d16-0`

### PR DESCRIPTION
**Pull request by VincentDondain**

### CODE REVIEW

**Comment by rolfbjarne** [comment]
build

File: Make.config [hunk 1]
```diff
@@ -69,17 +69,16 @@ MIN_SHARPIE_VERSION=3.4.23
 MAX_SHARPIE_VERSION=3.4.99
 MIN_SHARPIE_URL=https://bosstoragemirror.blob.core.windows.net/objective-sharpie/builds/4cde014216e8887375f9793d3a2607529833443b/440/76194/ObjectiveSharpie-3.4.23.pkg
 
-# Minimum OSX versions for building XI/XM
-MIN_OSX_BUILD_VERSION=10.13
-# Minimum OSX version for executing XI/XM tooling.
```
Comment (rolfbjarne) [inline Make.config]: This comment should stay.

**Comment by VincentDondain** [comment]
build

**Comment by VincentDondain** [comment]
We hit 32b dialog on the bot... https://github.com/xamarin/maccore/issues/1412

**Comment by chamons** [comment]
build

**Comment by VincentDondain** [comment]
Looks good to merge (:

**Pull request merged**